lib/minimagick/tool.rb

```diff
@@ -0,0 +1,16 @@
+# Hack to monkey-patch MiniMagick to always add the 'quiet'
+# option to every imagemagick command line.
+class MiniMagick::Tool
+  class_attribute :quiet_arg
+  self.quiet_arg = false
+
+  prepend(Module.new do
+    def command
+      if quiet_arg
+        [*executable, *(['-quiet'] + args)]
+      else
+        super
+      end
+    end
+  end)
+end
```

Comment:
I mean, if minimagick itself has a source file at it's own `./lib/minimagick/tool.rb`, then when something tries to load `Minimagick::Tool` (or `require "minimagick/tool"`), it may get out file, and _not_ the original. Our file will end up the sole implementing source for `Minimagick::Tool`, it won't be patching the original, it'll be the whole thing, cause it'll shadow the original from ever being loaded at all.